soh/soh/GameMenuBar.cpp

```diff
@@ -638,6 +638,9 @@ namespace GameMenuBar {
                         ImGui::EndMenu();
                     }
 
+                    UIWidgets::Spacer(0);
+                    ImGui::Separator();
+                    ImGui::Text("Meme Difficulty Options");
```

Comment:
You can remove the Spacer here and replace `ImGui::Separator()` with `UIWidgets::PaddedSeparator()`